doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1240 @@
+
+
+Multicore samples
+-----------------
+
+|no_changes_yet_note|
+
+nRF5340 samples
+---------------
+
+|no_changes_yet_note|
+
+Gazell samples
+--------------
+
+|no_changes_yet_note|
+
+Sensor samples
+--------------
+
+* Added :ref:`bme68x` sample to set up the BME68X gas sensor with the Bosch Sensor Environmental Cluster (BSEC) library.
+
+Zigbee samples
+--------------
+
+|no_changes_yet_note|
+
+Wi-Fi samples
+-------------
+
+* Added:
+
+  * :ref:`wifi_wfa_qt_app_sample` sample that demonstrates how to use the WFA QuickTrack (WFA QT) library needed for Wi-Fi Alliance QuickTrack certification.
+  * :ref:`wifi_shutdown_sample` sample that demonstrates how to configure the Wi-Fi driver to shut down the Wi-Fi hardware when the Wi-Fi interface is not in use.
+  * :ref:`wifi_twt_sample` sample that demonstrates how to establish TWT flow and transfer data conserving power.
+  * Support for the Wi-Fi driver to several upstream Zephyr networking samples.
+  * Enhancements to support device re-trimming process in the :ref:`wifi_radio_test` sample.
+  * Changes to the :ref:`wifi_scan_sample` sample to demonstrate usage of new scan APIs.
+
+Other samples
+-------------
+
+* Added the :ref:`802154_sniffer` sample.
+
+* Removed the Random hardware unique key sample.
+  The sample is redundant since its functionality is presented as part of the :ref:`hw_unique_key_usage` sample.
+
+* :ref:`radio_test` sample:
+
+  * Aligned the timer's configuration to the new nrfx API.
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+* Added :ref:`bme68x_iaq` to run the Bosch Sensor Environmental Cluster (BSEC) library in order to get Indoor Air Quality (IAQ) readings.
+
+Wi-Fi drivers
+-------------
+
+* Updated:
+
+  * TCP/IP checksum offload to enable by default for the nRF70 Series.
+  * Provision to change TX power ceilings using DTS file.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+* Added:
+
+  * :ref:`nrf_security` library, relocated from the sdk-nrfxlib repository to the :file:`subsys/nrf_security` directory.
+  * :ref:`network_core_monitor` library for monitoring the status of the nRF5340 processor's network core.
+
+Debug libraries
+---------------
+
+* :ref:`cpu_load` library:
+
+  * Updated by aligning the timer's configuration to the new nrfx API.
+
+Binary libraries
+----------------
+
+* :ref:`lib_bt_ll_acs_nrf53_readme` library:
+
+  * Added a limitation about the lack of support for the +20 dBm setting when :ref:`building the nRF5340 Audio application with the nRF21540 FEM support <nrf53_audio_app_adding_FEM_support>`.
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Updated to 3.3.0.
+    See the :ref:`liblwm2m_carrier_changelog` for detailed information.
+
+Bluetooth libraries and services
+--------------------------------
+
+* :ref:`bt_fast_pair_readme` library:
```

Comment:
@alstrzebonski, please also take a look at the Fast Pair library section in the NCS 2.5.0 release notes